lua/stylua-nvim.lua

```diff
@@ -98,4 +105,9 @@ M.format_range = function(start, stop, user_config)
   end
 end
 
+M.setup = function(opt)
```

Comment:
Since we're introducing a setup function here, what do you think about also looking for the other value in the config `error_display_strategy`?  This way users can start to just use this `setup` instead of passing the `user_config`  around. I'll deprecate that after.

Also can you add some documentation for this into the `stylua-nvim.txt` file in the LUA API section?

Reply:
Sounds good to me. Let me know what you think of the updates.